tests/integration-tests/README.md

```diff
@@ -861,3 +887,82 @@ def run_benchmarks(request, mpi_variants, test_datadir, instance, os, region, be
 
 * `IdentityFile` option in `ssh/config` will trigger a `str has no attribute extend` bug in the `fabric` package. 
 Please remove `IdentityFile` option from `ssh/config` before running the testing framework
+
+
+## Testing a scheduler plugin
```

Comment:
Do you want to mention the possibility to use a local plugin?

Reply:
it's done when describing the `scheduler-definition`. Were you thinking to something different?